src/crud.js

```diff
@@ -0,0 +1,39 @@
+const inputName = document.querySelector('.input-name');
+const inputScore = document.querySelector('.input-score');
+const ul = document.querySelector('.names-container');
+
+export const postUser = async () => {
+  const res = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/NGhCX4R3qCILaTqPLKM0/scores/',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: inputName.value,
+        score: inputScore.value,
+      }),
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    },
+  );
+  const data = await res.json();
+  return data;
+};
+
+// eslint-disable-next-line consistent-return
```

Comment:
- [Optional] Always endeavor to fix issues flagged by linters instead of disabling them. Linters analyze your code and flags programming errors. Always try to fix these errors